manifests/resource/server.pp

```diff
@@ -316,9 +320,24 @@
     }
   }
 
+  # If port range is defined, ignore any other $listen_port defined
+  if versioncmp($facts['nginx_version'], '1.15.10') < 0 {
+    $port = $listen_port
+
+    if $ipv6_enable == true {
```

Comment:
You don't need to update the code, but the following would work as well and is shorter:
```suggestion
    if $ipv6_enable {
```